lib/driver.js

```diff
@@ -30,7 +30,7 @@ let iDevice = function (...args) {
 class IosDriver extends BaseDriver {
   resetIos () {
     this.appExt = ".app";
-    _.defaults(this.opts, {
+    this.serverDetails = {
```

Comment:
just curious, why do we create a field on the object for this instead of just using a local variable? i'm all for having as few member fields as possible. i.e., if it's not keeping any essential state, let's not remember it.